tests/pdf_viewer/test_pdf_download.py

```diff
@@ -12,9 +12,18 @@ def add_prefs():
     return []
 
 
+@pytest.fixture()
```

Comment:
I see, so we override the regex in the original conftest to specify what we want to delete right? If that's the case I want to merge this into my MIME types PR to use

Reply:
yep!